src/CodeStyle/CSharp/CodeFixes/build/Microsoft.CodeAnalysis.CSharp.CodeStyle.props

```diff
@@ -0,0 +1,13 @@
+﻿<Project>
+  <!-- 
+    PropertyGroup to disable built-in CodeStyle analyzers from .NET SDK that have the identical IDE rules to those implemented in this package.
+    This props file should only be present in the analyzer NuGet package, it should **not** be inserted into the .NET SDK.
+    In future, if we need to insert a CodeStyle props file into the .NET SDK, we should move out the below property group into a separate props file
+    which is not inserted into the .NET SDK.
+
+    NOTE: Keep this file in sync with 'Microsoft.CodeAnalysis.VisualBasic.CodeStyle.props'
+  -->
+  <PropertyGroup>
+    <EnableStyleAnalyzers>false</EnableStyleAnalyzers>
```

Comment:
@jmarolf @mikadumont let me know what user facing property name we prefer to enable/disable code style build enforcement. `EnableStyleAnalyzers` is more in line with `EnableNetAnalyzers`, but I think `EnableStyleEnforcement` or `EnforceCodeStyleInBuild`, or some such property is more accurate as this property basically controls whether or not analyzers are enforced on build when escalated to warning/error. Setting this property to false does not disable CodeStyle analyzers in the IDE.

Reply:
@jmarolf @sharwell do you think we should also change the version of the package and assemblies to match the SDK analyzers, i.e. the SDK version itself like 5.0.0-rc1, 5.0.0-rc2, 5.0.0, and so on?

Reply:
I like `EnforceCodeStyleInBuild` better as a name. I'll change it in https://github.com/dotnet/sdk/pull/12968.

Reply:
@mikadumont Can you also confirm if `EnforceCodeStyleInBuild` looks good to you?

Reply:
> do you think we should also change the version of the package and assemblies to match the SDK analyzers

I don't know. I could see the argument that the version number should match the compiler version or that the version should match the version of the SDK that it shipped in. If @sharwell doesn't have an opinion then I would lean towards having it match the SDK for user-facing consistency.